### PR TITLE
Set default locale to Khmer (km)

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/KH-215)

### CODE REVIEW

File: configs/openmrs_config/globalproperties/i18n.xml [hunk 1]
```diff
@@ -4,5 +4,10 @@
             <property>locale.allowed.list</property>
             <value>en, km</value>
         </globalProperty>
+        <globalProperty>
+        <property>default_locale</property>
```
Comment (Ruhanga) [inline configs/openmrs_config/globalproperties/i18n.xml:8]: Some slight formatting needed here @kazlaw.

**Review by Ruhanga — CHANGES_REQUESTED** [review]
Thanks @kazlaw. Just a small change to make and we are good.

File: configs/openmrs_config/globalproperties/i18n.xml (hunk 1 above)
Comment (kazlaw) [reply configs/openmrs_config/globalproperties/i18n.xml:8]: Noted let me work on it.

Comment (kazlaw) [reply configs/openmrs_config/globalproperties/i18n.xml]: @Ruhanga , I have made adjustments. Is the latest commit good enough?

**Review by Ruhanga — APPROVED** [review]
LGTM

**Comment by Ruhanga** [comment]
@rbuisson could you merge this in? Thanks.

**Pull request merged**